test/analytics/test_gitlab_ci.py

```diff
@@ -0,0 +1,55 @@
+import yaml
+
+from zoo.analytics.tasks import gitlab_ci as uut
+
+YAML = """\
+stages:
+  - build
+  - test
+  - release
+  - deploy
+
+image: docker:19.03
+
+include:
+  - 'https://ci-files.skypicker.com/templates/build/coala.yml'
+  - 'https://ci-files.skypicker.com/templates/build/black.yml'
+  - 'https://ci-files.skypicker.com/templates/build/docker_build.yml'
+  - 'https://ci-files.skypicker.com/templates/release/sonarqube_scan.yml'
+  - 'https://ci-files.skypicker.com/templates/deploy/.crane.yml'
+
+black:  
+  stage: lint-and-build
+  image: kiwicom/black:19.3b0
+  script:
+    - black -l 120 --check --fast docs kw scripts test run.py setup.py
+deploy_production:
+  extends: .crane
+  variables:
+    CRANE_SLACK_CHANNEL: the-zoo
+  environment:
+    name: production
+    url: https://zoo.skypicker.com
+  when: manual
+"""
+
+
+def test_parse_docker_images():
+    expected_images = [("docker", "19.03"), ("kiwicom/black", "19.3b0")]
+    parsed_yaml = yaml.safe_load(YAML)
+
+    images = uut.parse_docker_images(parsed_yaml)
+    for image in images:
+        assert image.name, image.version in expected_images
```

Comment:
I think that the correct assertion would be that **only** those images are returned, as it might be returning more things.